packages/docs-reanimated/src/components/AvailableFrom/index.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import clsx from 'clsx';
+import { useColorMode } from '@docusaurus/theme-common';
+
+import styles from './styles.module.css';
+import Danger from '/static/img/danger.svg';
+import DangerDark from '/static/img/danger-dark.svg';
+
+interface Props {
+  version?: string;
+  isFull?: boolean;
+}
+
+export default function AvailableFrom({ version, isFull }: Props) {
+  const { colorMode } = useColorMode();
+  const isLightMode = colorMode === 'light';
+  const badgeThemeStyles = isLightMode ? styles.light : styles.dark;
+
+  const getBadgeContent = () => {
+    if (isFull) {
+      return (
+        <>
+          <div className={styles.infoIcon}>
+            {isLightMode ? <Danger /> : <DangerDark />}
+          </div>
+          <span>
+            Available from Reanimated{' '}
+            <span className={styles.version}>v{version}</span>
```

Comment:
Let's drop `v` prefix here for better readability
```suggestion
            <span className={styles.version}>{version}</span>
```

Reply:
Should we drop `v` prefix for both full version and normal?

Reply:
I think yes.